app/views/organizations/organization_profiles/_form.html.erb

```diff
@@ -71,6 +66,9 @@
 
       <!-- Third Column -->
       <div class="col-lg-6">
+        <%= form.text_field :facebook_url, label: "Facebook", class: 'form-control' %>
```

Comment:
can we add a placeholder value for each? Something like https://www.example.com just so they know its for URLs and to include the protocol.